qualtran/cirq_interop/_cirq_interop.py

```diff
@@ -43,11 +44,16 @@
     SoquetT,
 )
 from qualtran._infra.composite_bloq import _binst_to_cxns
+from qualtran.bloqs.util_bloqs import Allocate, Free
 
 CirqQuregT = NDArray[cirq.Qid]
 CirqQuregInT = Union[NDArray[cirq.Qid], Sequence[cirq.Qid]]
 
 
+def signature_from_cirq_registers(registers: Iterable[cirq_ft.Register]) -> 'Signature':
```

Comment:
elsewhere in this module we import `cirq_ft.Register` as `LegacyRegister`. Please be consistent throughout

Reply:
add to `cirq_interop/__init__.py`

Reply:
changed to `cirq_ft.Register` throughout.